play.py

```diff
@@ -164,14 +160,11 @@
         pr = Api.api.get_doc("PreRechnung", pr['name'])  # reload
         pinv2 = prerechnung.read_and_transfer(pr, check_dup=False)
         pinv2 = Api.api.get_doc("Purchase Invoice", pinv2.name)
-        validate(
-            instance=pinv1,
-            schema=JSON1_DATA_SCHEMA,
-        )
-        pr['json1'] = json.dumps(pinv1)
-        pr['json2'] = json.dumps(pinv2)
-        pr['doctype'] = 'PreRechnung'
-        Api.api.update(pr)
+        if validate_json1(pinv1):
```

Comment:
There should be a function for just validating all PreRechnungs, without modifying anything,

Reply:
There should be a function for validating all PreRechnungs.

Reply:
Added